cpp/config/Make.rules

```diff
@@ -97,7 +97,10 @@ endif
 # the cpp11 name to the target directory if building outside the build
 # directory.
 #
-cpp11_cppflags          = -DICE_CPP11_MAPPING -std=c++11
+cpp11_cppflags          = -DICE_CPP11_MAPPING
+ifneq ("OK", $(shell $(CC) -E config/cplusplus_check.cpp 2> /dev/null && echo "OK"))
+cpp11_cppflags          := $(cpp11_cppflags) -std=c++11
```

Comment:
Should this be indented?

Reply:
I think we do the same in other places, we do not indent statements inside `ifeq`
```
ifeq ($(filter all cpp,$(ICE_BIN_DIST)),)
$1_slicecompiler        := slice2cpp
```

Reply:
Ok